pkg/util/exec/exec.go

```diff
@@ -169,10 +169,16 @@ func startCommand(debug bool, command string, arg ...string) (*exec.Cmd, io.Read
 	logCommand(debug, command, arg...)
 
 	cmd := exec.Command(command, arg...)
-	stdout, _ := cmd.StdoutPipe()
-	stderr, _ := cmd.StderrPipe()
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		logger.Warningf("failed to open stdout pipe: %+v", err)
```

Comment:
Have you seen any occurrences of this? Definitely good to log it though so we can see when it does fail.

Reply:
I have not, but I added these in while I was poking around for possible issues. These effectively map to pipe2 syscall, so failure would be likely resource depletion of some sort. Pretty unlikely.